src/ray/core_worker/test/actor_task_submitter_test.cc

```diff
@@ -654,8 +635,28 @@ TEST_P(ActorTaskSubmitterTest, TestActorRestartFailInflightTasks) {
   const auto death_cause = CreateMockDeathCause();
   submitter_.DisconnectActor(
       actor_id, 1, /*dead=*/false, death_cause, /*is_restartable=*/true);
+  ASSERT_EQ(submitter_.NumInflightTasks(actor_id), 0);
+  // RPC callbacks are delayed.
+  ASSERT_EQ(worker_client_->callbacks.size(), 2);
 
-  // The task replies are now received. Since the tasks are already failed, they will not
+  // Task2 and task3 are retried.
```

Comment:
```suggestion
  // Submit retries for task2 and task3.
```

Reply:
"Task2 and task3 are retried." sounds like you're checking that that was done by the interface we're testing. That's not the case, the test is retrying them to validate the behavior that happens in response.